<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Report</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icon.ico</normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_post">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>MKI Report</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icons8-open-file-folder-96.png</normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icons8-open-file-folder-96.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>66</width>
      <height>66</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_post_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>321</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>JTS2 Report</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icons8-open-file-folder-96.png</normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icons8-open-file-folder-96.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>66</width>
      <height>66</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_post_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>321</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>JTS1 Report</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icons8-open-file-folder-96.png</normaloff>../CycleCount/Cycle Counting/REPORT/Assets/icons8-open-file-folder-96.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>66</width>
      <height>66</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
